src/plot_api/accessibility.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+import c2mChart from "chart2music";
+
+export function enable (gd) {
+
+    const c2mData = {};
+    const labels = [];
+    
+    const fullData = gd._fullData;
+
+    for(var i = 0; i < fullData.length; i++) {
+        var trace = fullData[i] ?? {};
+        var {type, x = [], y = [], name = i, text = []} = trace;
+        if(type === 'scatter') {
+            var traceData = [];
+            if ('y' in trace) {
+                for(var p = 0; p < y.length; p++) {
+                    traceData.push(
+                        {
+                            x: x.length > 0 ? x[p] : p,
+                            y: y[p],
+                            label: text[p] ?? p
+                        })
+                }
+                c2mData[name] = traceData;
+                labels.push(name);
+            }
+        }
+        else {
+            // 'Accessibility not implemented for trace type: ' + trace.type
+            return;
+        }
+    }
+    
+    var closed_captions = document.createElement('div');
+    closed_captions.id = 'cc';
+    closed_captions.className = 'closed_captions';
+    gd.appendChild(closed_captions);
+
+    const {
+        title: {text: title_text = ''} = {},
+        xaxis: {title: {text: xaxis_text = ''} = {}} = {},
+        yaxis: {title: {text: yaxis_text = ''} = {}} = {},
+    } = gd._fullLayout;
+    
+    c2mChart({
+        title: title_text,
+        type: "line",
```

Comment:
If this PR is for scatter plots, you probably want to have `type: "scatter"`.

Also note that, if you want to support more chart types, chart2music supports an array. So, if you have 2 traces, 1 scatter and 1 line, you could convert that to chart2music's `type: ["line", "scatter"]`.

Reply:
Thank you. I think I'll end up extending the scope to as many types as we can

Reply:
Thank you for supporting our project, @julianna-langston